zanata-war/src/main/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -196,6 +196,18 @@
     <to last="true">/profile/merge_account.seam$1</to>
   </rule>
 
+  <rule>
+    <from casesensitive="true">^/profile/view/([^/]+)(/([^?]*)\??(.*))?$</from>
```

Comment:
Ah ok... it's just that the project page in particular has a more complicated regexp because it basically allows any urls of the form `/project/view/myproject/../..` where you can add as many parameters as you want. These parameters will then get parsed by javascript to show specific portions of the UI.
If we don't need this functionality in this page, we can use a simpler regex. That's all I was suggesting.